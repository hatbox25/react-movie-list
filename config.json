{
  "name": "",
  "css_prefix_text": "icon-",
  "css_use_suffix": false,
  "hinting": true,
  "units_per_em": 1000,
  "ascent": 850,
  "glyphs": [
    {
      "uid": "b3706dfdec3c758786cb9ae8651b336b",
      "css": "home",
      "code": 59392,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M471.9 165L230.8 391.3 228 394.4C221.8 402 218.8 412.2 218.8 422.5V773.5L218.9 777.8C220.8 798.8 236.3 812.5 258.9 812.5H741.1L745.5 812.3C767.1 810.5 781.3 795.4 781.3 773.5V422.5L781.1 418.6C780.3 408.2 775.9 398.2 765.2 391.3L524.1 165 520.3 162.5C505 153.4 486.7 154.2 471.9 165ZM265.6 765.6V422.9L498 204.8 734.4 426.7V765.6H265.6Z",
        "width": 1000
      },
      "search": [
        "home.icon"
      ]
    },
    {
      "uid": "44466bf168829eeb580f3886da6b3fa2",
      "css": "analysis",
      "code": 59393,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M179.7 234.4C166.7 234.4 156.3 223.9 156.3 210.9 156.3 198 166.7 187.5 179.7 187.5H218.8 781.3 820.3C833.3 187.5 843.8 198 843.8 210.9 843.8 223.9 833.3 234.4 820.3 234.4H781.3V546.9C781.3 607.3 732.3 656.3 671.9 656.3H577.1L693.4 772.5C702.5 781.6 702.5 796.5 693.4 805.6 684.2 814.8 669.4 814.8 660.2 805.6L510.8 656.3H489.2L339.8 805.6C330.6 814.8 315.8 814.8 306.7 805.6 297.5 796.5 297.5 781.6 306.7 772.5L422.9 656.3H328.1C267.7 656.3 218.8 607.3 218.8 546.9V234.4H179.7ZM265.6 234.4V537.3C265.6 577.1 298.4 609.4 338.9 609.4H661.1C701.6 609.4 734.4 577.1 734.4 537.3V234.4H265.6ZM500 562.5C517.3 562.5 531.3 548.5 531.3 531.3V343.6C531.3 326.4 517.3 312.4 500 312.4 482.7 312.4 468.8 326.4 468.8 343.6V531.3C468.8 548.5 482.7 562.5 500 562.5ZM375 562.5C392.3 562.5 406.3 548.5 406.3 531.3V500C406.3 482.7 392.3 468.8 375 468.8 357.7 468.8 343.8 482.7 343.8 500V531.3C343.8 548.5 357.7 562.5 375 562.5ZM656.3 531.3C656.3 548.5 642.3 562.5 625 562.5 607.7 562.5 593.8 548.5 593.8 531.3V468.8C593.8 451.5 607.7 437.5 625 437.5 642.3 437.5 656.3 451.5 656.3 468.8V531.3Z",
        "width": 1000
      },
      "search": [
        "analysis.icon"
      ]
    },
    {
      "uid": "46fc4a7cc3df053111a70d29ba983103",
      "css": "facebook",
      "code": 59396,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M408.1 812.5V527.8H312.5V416.4H408.1V334.4C408.1 239.3 466.2 187.5 551 187.5 591.6 187.5 646.1 190.5 656.2 191.9V291.3H578.2C532.1 291.3 523.1 313.2 523.1 345.4V416.4H653L638.6 527.8H523.1V812.5H408.1Z",
        "width": 1000
      },
      "search": [
        "facebook"
      ]
    },
    {
      "uid": "748fef21ab08c29e6b5d2c112aac7757",
      "css": "instagram",
      "code": 59397,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M813.9 298.6C821.9 318.9 827.4 342.4 828.9 376.6 830.4 410.9 830.5 421.8 830.5 509.1 830.5 596.5 830.1 607.4 828.6 641.7 827.1 675.9 821.6 699.2 813.7 719.7 797 762.9 762.8 797.1 719.6 813.8 699.2 821.7 675.7 827.3 641.6 828.8 607.3 830.3 596.3 830.6 509 830.6 421.7 830.6 410.8 830.3 376.5 828.8 342.3 827.3 318.9 821.7 298.4 813.8 277.1 805.8 257.6 793.1 241.5 776.8 225.2 760.7 212.5 741.3 204.3 719.7 196.4 699.4 190.9 675.9 189.4 641.7 187.9 607.4 187.5 596.5 187.5 509.1 187.5 421.8 187.9 410.9 189.5 376.5 191 342.3 196.5 318.9 204.5 298.4 212.5 277.1 225.2 257.6 241.5 241.5 257.6 225.1 277 212.5 298.6 204.3 318.9 196.4 342.4 190.9 376.6 189.4 410.9 187.9 421.8 187.5 509.1 187.5 596.5 187.5 607.4 187.9 641.8 189.5 676 191 699.4 196.5 719.8 204.5 741.2 212.5 760.7 225.2 776.8 241.5 793.2 257.6 805.8 277 813.9 298.6ZM759.9 698.9C764.3 687.5 769.6 670.6 771 639.2 772.5 605.3 772.9 595.2 772.9 509.4 772.9 423.6 772.5 413.4 771 379.5 769.6 348.2 764.3 331.1 759.9 319.8 754.8 305.9 746.6 293.3 735.9 282.9 725.6 272.2 712.9 264 699 258.9 687.7 254.5 670.7 249.2 639.3 247.8 605.5 246.3 595.3 245.9 509.4 245.9 423.6 245.9 413.4 246.3 379.5 247.8 348.2 249.2 331.1 254.5 319.8 258.9 305.9 264 293.3 272.2 282.9 282.9 272.1 293.2 263.9 305.9 258.7 319.8 254.3 331.1 249.1 348.1 247.7 379.5 246.2 413.3 245.8 423.6 245.8 509.4 245.8 595.2 246.2 605.4 247.7 639.3 249.1 670.6 254.3 687.7 258.7 699 263.9 712.9 272.1 725.5 282.7 735.9 293 746.6 305.7 754.8 319.7 759.9 331 764.3 347.9 769.6 379.4 771 413.2 772.5 423.5 772.9 509.3 772.9 595.1 772.9 605.3 772.5 639.2 771 670.5 769.6 687.5 764.3 698.9 759.9 726.9 749.1 749.1 726.9 759.9 698.9ZM343.9 509.1C343.9 417.9 417.9 343.9 509.1 343.9 600.4 343.9 674.4 417.9 674.4 509.1 674.4 600.4 600.4 674.4 509.1 674.4 417.9 674.4 343.9 600.4 343.9 509.1ZM402 509.1C402 568.3 450 616.3 509.1 616.3 568.3 616.3 616.3 568.3 616.3 509.1 616.3 450 568.3 402 509.1 402 450 402 402 450 402 509.1ZM719.5 337.4C719.5 358.7 702.2 376 680.9 376 659.6 376 642.3 358.7 642.3 337.4 642.3 316.1 659.6 298.8 680.9 298.8 702.2 298.8 719.5 316.1 719.5 337.4Z",
        "width": 1000
      },
      "search": [
        "instagram"
      ]
    },
    {
      "uid": "eb9e163ef54a4963e54d70d4ac1f444b",
      "css": "google-analytics",
      "code": 59398,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M677.1 156.3C642.6 156.3 614.6 184.2 614.6 218.8V385.4H442.7C411.1 385.4 385.4 411.1 385.4 442.7V614.6H213.5C181.9 614.6 156.3 640.2 156.3 671.9V786.5C156.3 818.1 181.9 843.8 213.5 843.8H614.6 700.5 781.3C815.8 843.8 843.8 815.8 843.8 781.3V218.8C843.8 184.2 815.8 156.3 781.3 156.3H677.1ZM614.6 428.4V800.8H213.5C205.6 800.8 199.2 794.4 199.2 786.5V671.9C199.2 664 205.6 657.6 213.5 657.6H428.4V442.7C428.4 434.8 434.8 428.4 442.7 428.4H614.6Z",
        "width": 1000
      },
      "search": [
        "ga"
      ]
    },
    {
      "uid": "8100c204347ecc08b55409f530b85d09",
      "css": "google-adwords",
      "code": 59399,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M623.6 267.3L859.2 651.5C889.1 700.3 875.3 765 828.2 796.1 781.2 827.2 718.8 812.8 688.9 764L531.3 506.9 373.6 764 370.3 769.1C350 798.3 317.5 813.6 284.8 812.4 283.6 812.5 282.4 812.5 281.3 812.5 229.5 812.5 187.5 770.5 187.5 718.8 187.5 716.9 187.6 715.1 187.7 713.4 186.5 692.3 191.5 670.6 203.3 651.5L438.9 267.3 442.2 262.2C463.2 231.9 497.3 216.5 531.3 219 567 216.4 603 233.7 623.6 267.3ZM437.6 329L253.6 629.1C262.4 626.4 271.6 625 281.3 625 328 625 366.7 659.2 373.8 703.9L512.9 477 453.3 379.8C443.6 364 438.5 346.5 437.6 329Z",
        "width": 1000
      },
      "search": [
        "gads"
      ]
    },
    {
      "uid": "e8a8a083e56f3a71907d54a0645b37ee",
      "css": "coin",
      "code": 59400,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M500 0C223.9 0 0 223.9 0 500S223.9 1000 500 1000 1000 776.1 1000 500 776.1 0 500 0ZM643.6 697.1C621.8 723.7 593.4 741.6 560.3 750.6 545.9 754.5 539.3 762.1 540.1 777.2 540.7 792 540.1 806.6 539.9 821.4 539.9 834.6 533.1 841.6 520.2 842 511.7 842.2 503.3 842.4 494.9 842.4 487.4 842.4 480 842.4 472.6 842.2 458.6 842 452.1 834 452.1 820.4 451.9 809.7 451.9 798.8 451.9 788.1 451.6 764.2 450.8 763.4 428 759.7 398.8 754.9 370 748.4 343.2 735.4 322.2 725.1 320 720 325.9 697.7 330.5 681.3 335 664.8 340.1 648.6 343.8 636.6 347.3 631.3 353.7 631.3 357.4 631.3 362.1 633.1 368.5 636.4 398.1 651.9 429.6 660.5 462.8 664.6 468.3 665.2 473.9 665.6 479.4 665.6 494.9 665.6 509.9 662.8 524.5 656.4 561.3 640.3 567.1 597.7 536 572.2 525.5 563.6 513.4 557.2 500.8 551.6 468.5 537.4 435 526.7 404.5 508.4 355.1 478.8 323.9 438.3 327.6 378.2 331.7 310.3 370.2 267.9 432.5 245.3 458.2 236 458.4 236.2 458.4 209.5 458.4 200.4 458.2 191.4 458.6 182.1 459.3 161.9 462.6 158.4 482.7 157.8 485 157.8 487.4 157.8 489.7 157.8 493.6 157.8 497.5 157.8 501.4 157.8 503.1 157.8 504.7 157.8 506.2 157.8 544.4 157.8 544.4 159.5 544.7 200.8 544.9 231.3 544.9 231.3 575.1 236 598.4 239.7 620.4 246.5 641.8 256 653.5 261.1 658 269.3 654.3 281.9 649 300.4 643.8 319.1 638.1 337.4 634.4 348.6 630.9 353.7 624.3 353.7 620.6 353.7 616 352.3 610.3 349.4 580.7 335 549.6 328 517.1 328 513 328 508.6 328.2 504.5 328.4 494.9 329 485.4 330.2 476.3 334.2 444.2 348.1 439.1 383.5 466.5 405.3 480.2 416.5 496.1 424.3 512.3 431.1 540.7 442.8 569.1 454.1 596.1 468.9 680.9 516.3 703.9 623.9 643.6 697.1Z",
        "width": 1000
      },
      "search": [
        "coin"
      ]
    },
    {
      "uid": "8234b26b0b30186954b6e02ab10e926c",
      "css": "person",
      "code": 59403,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M500 500C592.1 500 666.7 425.4 666.7 333.3S592.1 166.7 500 166.7 333.3 241.3 333.3 333.3 407.9 500 500 500ZM500 583.3C388.8 583.3 166.7 639.2 166.7 750V833.3H833.3V750C833.3 639.2 611.3 583.3 500 583.3Z",
        "width": 1000
      },
      "search": [
        "person-24px"
      ]
    },
    {
      "uid": "3e352432d0ff046cf540957ba8853194",
      "css": "calendar",
      "code": 59404,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M791.7 166.7H750V83.3H666.7V166.7H333.3V83.3H250V166.7H208.3C162.1 166.7 125 204.2 125 250V833.3A83.3 83.3 0 0 0 208.3 916.7H791.7A83.3 83.3 0 0 0 875 833.3V250A83.3 83.3 0 0 0 791.7 166.7M791.7 833.3H208.3V416.7H791.7V833.3M791.7 333.3H208.3V250H791.7V333.3Z",
        "width": 1000
      },
      "search": [
        "calendar-24px"
      ]
    },
    {
      "uid": "4eaa64771df195f3f45ba414d1303a3b",
      "css": "config",
      "code": 59401,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M784.4 390.7L702.5 370.6C698 356.2 692.3 342.3 685.3 328.7L727.1 259C732.1 250.8 730.8 240.5 724.3 234L666 175.7C659.5 169.2 649.2 167.9 641 172.9L571.3 214.7C557.7 207.7 543.8 202 529.4 197.5L509.7 115.6C507.3 106.6 499.1 100 489.6 100L407.6 100C398.2 100 390 106.6 387.5 115.6L367.8 197.5C353.5 202 339.5 207.7 326 214.7L256.3 172.9C248.1 167.9 237.8 169.2 231.3 175.7L173 234C166.4 240.5 165.2 250.8 170.1 259L212 328.7C205 342.3 199.3 356.2 194.7 370.6L115.6 390.7C106.6 392.7 100 400.9 100 410.4L100 492.4C100 501.8 106.6 510 115.6 512.1L194.7 532.2C199.3 546.5 205 560.5 212 574L170.1 643.7C165.2 651.9 166.4 662.2 173 668.8L231.3 727C237.8 733.6 248.1 734.8 256.3 729.9L326 688C339.5 695 353.5 700.7 367.8 705.3L387.5 784.4C390 793.4 398.2 800 407.6 800L489.6 800C499.1 800 507.3 793.4 509.7 784.4L529.4 705.3C543.8 700.7 557.7 695 571.3 688L641 729.9C649.2 734.8 659.5 733.6 666 727L724.3 668.8C730.8 662.2 732.1 651.9 727.1 643.7L685.3 574C692.3 560.5 698 546.5 702.5 532.2L784.4 512.1C793.4 510 800 501.8 800 492.4L800 410.4C800 400.9 793.4 392.7 784.4 390.7ZM448.6 553.9C392 553.9 346.1 508 346.1 451.4 346.1 394.8 392 348.8 448.6 348.8 505.2 348.8 551.2 394.8 551.2 451.4 551.2 508 505.2 553.9 448.6 553.9Z",
        "width": 1000
      },
      "search": [
        "config"
      ]
    },
    {
      "uid": "1d35198f5190ec004dd4ec742fbe19ca",
      "css": "right",
      "code": 59405,
      "src": "mfglabs"
    },
    {
      "uid": "693a43eb91c4c7e1eaddb4f7ab51d1b0",
      "css": "comment",
      "code": 59402,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M916.2 166.7C916.2 120.8 879.2 83.3 833.3 83.3H166.7C120.8 83.3 83.3 120.8 83.3 166.7V666.7C83.3 712.5 120.8 750 166.7 750H750L916.7 916.7 916.2 166.7ZM750 583.3H250V500H750V583.3ZM750 458.3H250V375H750V458.3ZM750 333.3H250V250H750V333.3Z",
        "width": 1000
      },
      "search": [
        "comment-24px"
      ]
    },
    {
      "uid": "f34e197f64e083c53a05357726763edf",
      "css": "filter-menu",
      "code": 59406,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M458.3 458.3L698.3 150.8A41.7 41.7 0 0 0 691.3 92.5 41.7 41.7 0 0 0 666.7 83.3H83.3A41.7 41.7 0 0 0 57.5 92.5 41.7 41.7 0 0 0 50.4 150.8L291.7 458.3V702.9A41.7 41.7 0 0 0 303.8 737.5L387.1 820.8A41.7 41.7 0 0 0 445.8 820.8 41.7 41.7 0 0 0 458.3 786.3V458.3M541.7 666.7L750 875 958.3 666.7Z",
        "width": 1000
      },
      "search": [
        "filter-menu"
      ]
    },
    {
      "uid": "d867a35313897ef1d34d01789a10b088",
      "css": "file-export",
      "code": 59407,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M250 83.3C203.7 83.3 166.7 120.8 166.7 166.7V833.3A83.3 83.3 0 0 0 250 916.7H750A83.3 83.3 0 0 0 833.3 833.3V333.3L583.3 83.3M541.7 145.8L770.8 375H541.7M372.1 509.2H666.7V803.7L578.3 715.4 460.4 833.3 342.5 715.4 460.4 597.9",
        "width": 1000
      },
      "search": [
        "file-export"
      ]
    },
    {
      "uid": "3caf240de1ec0951264375c2f59470e8",
      "css": "account",
      "code": 59408,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M500 800C395.8 800 303.8 746.7 250 666.7 251.3 583.3 416.7 537.5 500 537.5 583.3 537.5 748.7 583.3 750 666.7 696.3 746.7 604.2 800 500 800M500 208.3C569.2 208.3 625 264.2 625 333.3 625 402.5 569.2 458.3 500 458.3 430.8 458.3 375 402.5 375 333.3 375 264.2 430.8 208.3 500 208.3M500 83.3C270 83.3 83.3 270 83.3 500 83.3 730 270 916.7 500 916.7 730 916.7 916.7 730 916.7 500 916.7 269.6 729.2 83.3 500 83.3Z",
        "width": 1000
      },
      "search": [
        "account-circle"
      ]
    },
    {
      "uid": "3327862df439988139b61814143b7a42",
      "css": "cancel-circled",
      "code": 59409,
      "src": "typicons"
    },
    {
      "uid": "6cc7af3e5b55720bcb6ef68372ce24be",
      "css": "facebook-official",
      "code": 62000,
      "src": "fontawesome"
    },
    {
      "uid": "09da8b02aa377dcb7bf46e2f52e2a739",
      "css": "search-web",
      "code": 59410,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M645.8 583.3L854.2 791.7 791.7 854.2 583.3 645.8V612.9L572.1 601.3C524.6 642.1 462.9 666.7 395.8 666.7A270.8 270.8 0 0 1 125 395.8 270.8 270.8 0 0 1 395.8 125 270.8 270.8 0 0 1 666.7 395.8C666.7 462.9 642.1 524.6 601.3 572.1L612.9 583.3H645.8M395.8 187.5L372.9 188.8C362.9 210.4 347.5 247.1 336.3 291.7H455.4C444.2 247.1 428.7 210.4 418.8 188.8 411.2 187.5 403.7 187.5 395.8 187.5M576.3 291.7C551.7 248.7 512.1 215.4 464.6 199.2 474.6 221.2 487.5 253.3 497.1 291.7H576.3M215.4 291.7H294.6C304.2 253.3 317.1 221.2 327.1 199.2 279.6 215.4 240 248.7 215.4 291.7M187.5 395.8C187.5 416.7 190.8 438.7 197.1 458.3H286.3L281.3 395.8 286.3 333.3H197.1C190.8 352.9 187.5 375 187.5 395.8M594.6 458.3C600.8 438.7 604.2 416.7 604.2 395.8 604.2 375 600.8 352.9 594.6 333.3H505.4C508.8 354.2 510.4 375 510.4 395.8 510.4 416.7 508.8 437.5 505.4 458.3H594.6M327.9 333.3L322.9 395.8 327.9 458.3H463.8C467.1 437.5 468.8 416.7 468.8 395.8 468.8 375 467.1 354.2 463.8 333.3H327.9M395.8 604.2C403.3 604.2 410.8 604.2 417.9 602.9 428.3 581.2 444.2 544.6 455.4 500H336.3C347.5 544.6 363.3 581.2 373.8 602.9L395.8 604.2M576.3 500H497.1C487.5 538.3 474.6 570.4 464.6 592.5 512.1 576.3 551.7 542.9 576.3 500M215.4 500C240 542.9 279.6 576.3 327.1 592.5 317.1 570.4 304.2 538.3 294.6 500H215.4Z",
        "width": 1000
      },
      "search": [
        "search-web"
      ]
    },
    {
      "uid": "e9ece780c32ace811dcda6c736cacdce",
      "css": "plus",
      "code": 59411,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M791.7 541.7H541.7V791.7H458.3V541.7H208.3V458.3H458.3V208.3H541.7V458.3H791.7V541.7Z",
        "width": 1000
      },
      "search": [
        "plus"
      ]
    },
    {
      "uid": "d7e63cf2530f90462690b29d24f146db",
      "css": "link",
      "code": 59412,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M162.5 500C162.5 428.7 220.4 370.8 291.7 370.8H458.3V291.7H291.7A208.3 208.3 0 0 0 83.3 500 208.3 208.3 0 0 0 291.7 708.3H458.3V629.2H291.7C220.4 629.2 162.5 571.3 162.5 500M333.3 541.7H666.7V458.3H333.3V541.7M708.3 291.7H541.7V370.8H708.3C779.6 370.8 837.5 428.7 837.5 500 837.5 571.3 779.6 629.2 708.3 629.2H541.7V708.3H708.3A208.3 208.3 0 0 0 916.7 500 208.3 208.3 0 0 0 708.3 291.7Z",
        "width": 1000
      },
      "search": [
        "link"
      ]
    },
    {
      "uid": "41087bc74d4b20b55059c60a33bf4008",
      "css": "edit",
      "code": 59413,
      "src": "fontawesome"
    },
    {
      "uid": "bbfb51903f40597f0b70fd75bc7b5cac",
      "css": "trash",
      "code": 61944,
      "src": "fontawesome"
    },
    {
      "uid": "3bd18d47a12b8709e9f4fe9ead4f7518",
      "css": "arrows-ccw",
      "code": 59414,
      "src": "entypo"
    },
    {
      "uid": "815503841e980c848f55e0271deacead",
      "css": "link-1",
      "code": 59415,
      "src": "entypo"
    },
    {
      "uid": "c619e17229d6c874dfc70be359b6b3f1",
      "css": "folder",
      "code": 61543,
      "src": "mfglabs"
    },
    {
      "uid": "1e89103762be72b0720eda0468e62b90",
      "css": "globe-1",
      "code": 59416,
      "src": "linecons"
    },
    {
      "uid": "e20215ad5fe27b6f29c2120ed894c114",
      "css": "send",
      "code": 59395,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M83.3 875L958.3 500 83.3 125V416.7L708.3 500 83.3 583.3V875Z",
        "width": 1000
      },
      "search": [
        "send"
      ]
    },
    {
      "uid": "giaskl13npzvuuz2ovy3vw8s0bewik08",
      "css": "up",
      "code": 59417,
      "src": "modernpics"
    },
    {
      "uid": "3iwbntpxmp0kvbvzaxrh8h30xw1l6zvn",
      "css": "down",
      "code": 59418,
      "src": "modernpics"
    },
    {
      "uid": "3632ca638d189a803048623204639ce8",
      "css": "marketplace",
      "code": 59394,
      "src": "custom_icons",
      "selected": true,
      "svg": {
        "path": "M772.2 356.7L772.3 356.8C772.2 356.8 772.2 356.8 772.3 356.8 772.3 356.9 772.3 356.9 772.3 357.1 772.3 403.1 736.9 439 695.3 439 671.1 439 649.2 427 634.9 407.7L619.8 387.3 604.7 407.7C590.4 427 568.5 439 544.3 439 520 439 498.1 427 483.8 407.7L468.8 387.3 453.7 407.7C439.4 427 417.4 439 393.2 439 369 439 347.1 427 332.8 407.7L317.7 387.3 302.6 407.7C288.3 427 266.4 439 242.2 439 200.6 439 165.2 403.1 165.2 357.1 165.2 356.9 165.2 356.9 165.3 356.8 165.3 356.8 165.3 356.8 165.3 356.8 165.3 356.8 165.3 356.8 165.3 356.8L165.3 356.7 201.7 210.9C201.7 210.8 201.7 210.8 201.7 210.8 204.2 201.2 212.8 194.5 222.7 194.5H714.9C724.8 194.5 733.4 201.3 735.8 210.8L735.8 210.8 772.2 356.7ZM695.3 498C667.9 498 641.8 489.7 619.8 474.4 575.7 505 512.8 505 468.8 474.4 424.7 505 361.8 505 317.7 474.4 295.7 489.7 269.6 498 242.2 498 222.4 498 203.7 493.4 186.8 485.5V699.5C186.8 721.7 204.8 739.7 227.1 739.7H388.2V578.6H549.3V739.7H710.5C732.7 739.7 750.7 721.7 750.7 699.5V485.5C733.8 493.4 715.1 498 695.3 498Z",
        "width": 1000
      },
      "search": [
        "marketplace.icon"
      ]
    },
    {
      "uid": "c83b2c6746c2489bd2bc6979239c8eba",
      "css": "web",
      "code": 59395,
      "src": "custom_icons",
      "selected": false,
      "svg": {
        "path": "M500 843.8C689.8 843.8 843.8 689.8 843.8 500 843.8 310.2 689.8 156.3 500 156.3 310.2 156.3 156.3 310.2 156.3 500 156.3 689.8 310.2 843.8 500 843.8ZM552.6 207.4C541.4 198 529.9 191.6 518.3 188.3V335.2C554.2 334.2 589 330.2 621.4 323.6 615.5 305.4 608.6 288.3 600.7 272.7 586.6 244.5 570.4 222.5 552.6 207.4ZM316.2 480.4C317.4 434.8 323.2 390.7 333 350.2 300.9 341.2 271.5 329.6 246.1 315.6 211.2 363.7 190.9 420.5 187.5 480.4H316.2ZM631.4 358.9C595.8 366.3 557.6 370.7 518.3 371.8V480.4H647.1C645.9 437.6 640.5 396.5 631.4 358.9ZM187.5 517.1H316.2C317.4 562.7 323.2 606.8 333 647.3 300.9 656.3 271.5 667.9 246.1 681.9 211.2 633.8 190.9 577 187.5 517.1ZM269.7 710.8C291.4 699.5 316 690 342.8 682.5 349.5 703.4 357.4 723.1 366.4 741.2 377.1 762.6 389 780.9 401.9 796.2 355.9 781.1 313.8 755.3 278.6 720.1 275.6 717 272.6 713.9 269.7 710.8ZM399.2 272.7C391.4 288.3 384.4 305.4 378.5 323.6 410.9 330.2 445.7 334.2 481.7 335.2V188.3C470 191.6 458.5 198 447.3 207.4 429.5 222.5 413.3 244.5 399.2 272.7ZM667 647.3C676.8 606.8 682.5 562.7 683.7 517.1H812.4C809 577 788.7 633.8 753.8 681.9 728.4 667.9 699 656.3 667 647.3ZM657.1 315C683.9 307.5 708.6 298 730.2 286.7 727.3 283.6 724.3 280.5 721.3 277.4 686.1 242.2 644 216.4 598 201.3 610.9 216.5 622.8 234.9 633.5 256.3 642.5 274.3 650.4 294 657.1 315ZM621.4 673.9C615.5 692.1 608.6 709.1 600.7 724.8 586.6 753 570.4 775 552.6 790 541.4 799.5 529.9 805.9 518.3 809.2V662.3C554.2 663.3 589 667.3 621.4 673.9ZM667 350.2C676.8 390.7 682.5 434.8 683.7 480.4H812.4C809 420.5 788.7 363.7 753.8 315.6 728.4 329.6 699 341.2 667 350.2ZM518.3 625.7V517.1H647.1C645.9 559.8 640.5 601 631.4 638.6 595.8 631.2 557.6 626.8 518.3 625.7ZM481.7 480.4V371.8C442.3 370.7 404.1 366.3 368.5 358.9 359.4 396.5 354.1 437.6 352.9 480.4H481.7ZM366.4 256.3C357.4 274.3 349.5 294 342.8 315 316 307.5 291.4 298 269.7 286.7 272.6 283.6 275.6 280.5 278.6 277.4 313.8 242.2 355.9 216.4 401.9 201.3 389 216.5 377.1 234.9 366.4 256.3ZM657.1 682.5C650.4 703.4 642.5 723.1 633.5 741.2 622.8 762.6 610.9 780.9 598 796.2 644 781.1 686.1 755.3 721.3 720.1 724.3 717 727.3 713.9 730.2 710.8 708.6 699.5 683.9 690 657.1 682.5ZM352.9 517.1H481.7V625.7C442.3 626.8 404.1 631.2 368.5 638.6 359.4 601 354.1 559.8 352.9 517.1ZM447.3 790C458.5 799.5 470 805.9 481.7 809.2V662.3C445.7 663.3 410.9 667.3 378.5 673.9 384.4 692.1 391.4 709.1 399.2 724.8 413.3 753 429.5 775 447.3 790Z",
        "width": 1000
      },
      "search": [
        "web.icon"
      ]
    },
    {
      "uid": "95af1f0e006518f4a2a3c4c1ff529940",
      "css": "send-24px",
      "code": 59411,
      "src": "custom_icons",
      "selected": false,
      "svg": {
        "path": "M2000 21000L23000 12000 2000 3000V10000L17000 12000 2000 14000V21000Z",
        "width": 1000
      },
      "search": [
        "send-24px"
      ]
    },
    {
      "uid": "c08f028f1c8b0fa888ce50bd9b0e3714",
      "css": "send-24px",
      "code": 59412,
      "src": "custom_icons",
      "selected": false,
      "svg": {
        "path": "M2000 21000L23000 12000 2000 3000V10000L17000 12000 2000 14000V21000Z",
        "width": 1000
      },
      "search": [
        "send-24px"
      ]
    },
    {
      "uid": "13fdf717e78f81a947ecfde2d05d2970",
      "css": "send-24px",
      "code": 59413,
      "src": "custom_icons",
      "selected": false,
      "svg": {
        "path": "M2000 21000L23000 12000 2000 3000V10000L17000 12000 2000 14000V21000Z",
        "width": 1000
      },
      "search": [
        "send-24px"
      ]
    }
  ]
}