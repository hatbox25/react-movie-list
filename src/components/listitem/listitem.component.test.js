import React from 'react';
import { shallow } from 'enzyme';

import Searchbar from './listitem.component';
import config from './listitem.component.config';

const props = {
  ...config.defaultProps,
  onClickImage: jest.fn(),
};

let component;

describe(config.displayName, () => {
  beforeEach(() => {
    component = shallow(<Searchbar {...props} />);
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  test('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
