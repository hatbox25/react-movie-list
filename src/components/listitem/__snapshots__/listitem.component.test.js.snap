// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should render correctly 1`] = `
<Item
  style={
    Object {
      "borderBottomColor": "#d4d4d4",
      "flexDirection": "column",
    }
  }
>
  <div
    onClick={[Function]}
  >
    <img
      alt="poster-list"
      height={200}
      src=""
    />
  </div>
  <Text>
     - ()
  </Text>
</Item>
`;
